### ISSUE TITLE
Heating bed no longer works after using ChangeAtZ on Cura 4.2.1

### ISSUE THREAD

**Issue by GodChronoss** [Ultimaker/Cura#6228]

**Comment by GodChronoss** [comment]
I fixed it. Thanks

**Comment by Ghostkeeper** [comment]
Maybe it'd be useful for people to know how you fixed it, in case they also encounter this problem.

**Comment by GodChronoss** [comment]
Oh sorry! I didn't know why the whole thing was deleted.

The problem with my 2 printers happened at the same time I was testing the ChangeAtZ script for the fist time. The gcode turned off my bed at the layer I wanted. The problem was the heating beds never tuned back on after that.

I was blaming the gcode for the failures. I was wrong!. It turned out it was the wiring on both my CR10s PRO's heating bed that got damaged while printing (one melted and the other one almost catches fire during troubleshooting). I ended up rewiring the beds with a 16 gauge teflon coated wire to fix both printers. 

I had it all explained in the post that I guess I deleted by accident. I don't know how it happened. I was really tired when I posted the fix. Sorry!
